en/middleware/mavlink.md

```diff
@@ -73,7 +73,7 @@ public:
     }
 
 private:
-    MavlinkOrbSubscription *_sub;
+    uORB::Subscription *_sub;
```

Comment:
```suggestion
    uORB::Subscription _sub{ORB_ID(ca_trajectory)};
```

There's more changes required further down:
- `_sub(_mavlink->add_orb_subscription(ORB_ID(ca_trajectory))),  // make sure you enter the name of your uORB topic here` -> remove
- `uint64_t _ca_traj_time;` -> remove
- `if (_sub->update(&_ca_traj_time, &_ca_trajectory)) {` -> `if (_sub.update(&_ca_trajectory)) {`